website/docs/error_handling.md

```diff
@@ -0,0 +1,49 @@
+---
+id: error_handler
+title: Error Handlers
+sidebar_label: Error Handlers
+---
+
+## Error Handlers
+`Core-Lib` error handlers contain decorator and function which can raise exceptions for various scenarios.
+
+###StatusCodeException
+Will raise an exception for the given `Status Code` ( to be passed as arguments ).

```

Comment:
let's try something like this 

`StatusCodeException` is the primary and single exception used by any `Core-Lib` 

it serves two primary purposes 
1. Unified way to handle errors while using `Core-Lib`
2. Reflect any error with a numeric status code
3. Bridge between library errors and HTTP Status code 

```Python
from http import HTTPStatus
from core_lib.error_handling.status_code_exception import StatusCodeException

....
raise StatusCodeException(HTTPStatus.BAD_REQUEST, 'Input parameter is invalid')
```